docs/flow-logic/error-handling/memory-errors.md

```diff
@@ -4,7 +4,9 @@ n8n doesn't restrict the amount of data each node can fetch and process. While t
 
 ## Identifying out of memory situations
 
-Error messages including **Problem running workflow**, **Connection Lost**, or **503 Service Temporarily Unavailable** suggest that an n8n instance has become unavailable. The execution list shows an **Unknown** status for the executions.
+n8n provides error messages that warn you in some out of memory situations. For example, messages such as **Execution stopped at this node (n8n may have run out of memory while executing it.)**.
```

Comment:
`it.).` looks a bit strange with the two dots...

Reply:
It does . . . I might ignore the general rule to match UI copy exactly and remove the one within the brackets.